src/InternalState.ts

```diff
@@ -17,58 +17,98 @@
  */
 
 export default class InternalState {
-  static devMode = false
+  private _devMode: boolean = false
 
-  static variablesChangedHandlers = []
-  static hasReceivedDiffs = false
+  private _variablesChangedHandlers: Function[] = []
+  private _hasReceivedDiffs: boolean = false
 
-  static startHandlers = []
-  static hasStarted = false
-  static startSuccessful = false
-  static variantDebugInfoEnabled: boolean = false
+  private _hasStarted: boolean = false
+  private _startHandlers: Function[] = []
+  private _startSuccessful: boolean = false
+  private _variantDebugInfoEnabled: boolean = false
 
-  static addStartResponseHandler(handler) {
-    InternalState.startHandlers.push(handler)
-    if (InternalState.hasStarted) {
-      handler(InternalState.startSuccessful)
+  public get devMode(): boolean {
+    return this._devMode
+  }
+
+  public set devMode(value: boolean) {
+    this._devMode = value
+  }
+
+  public get hasReceivedDiffs(): boolean {
+    return this._hasReceivedDiffs
+  }
+
+  public set hasReceivedDiffs(value: boolean) {
+    this._hasReceivedDiffs = value
+  }
+
+  public get hasStarted(): boolean {
+    return this._hasStarted
+  }
+
+  public set hasStarted(value: boolean) {
+    this._hasStarted = value
+  }
+
+  public get startSuccessful(): boolean {
+    return this._startSuccessful
+  }
+
+  public set startSuccessful(value: boolean) {
+    this._startSuccessful = value
+  }
+
+  public get variantDebugInfoEnabled(): boolean {
+    return this._variantDebugInfoEnabled
+  }
+
+  public set variantDebugInfoEnabled(value: boolean) {
```

Comment:
Why use get/set methods instead of public properties?

Reply:
For easier refactoring, but in this case it should be OK either way - I changed to public props.